docs/support.md

```diff
@@ -1,8 +1,9 @@
 ---
-title: Need help?
-layout: single
 id: support
-permalink: support.html
+title: Where To Get Support
+layout: community
+permalink: community/support.html
```

Comment:
Moving this URL but none of the rest that are going into the section?

Should we just go ahead and rename files to fit now as well?